docs/PROVISIONING_OPENSTACK.md

```diff
@@ -46,17 +46,7 @@ The following is just an example on how the `sample.osp.example.com` inventory c
 2) Edit `~/src/casl-ansible/inventory/sample.osp.example.com.d/inventory/group_vars/all.yml` to match your cloud provider/environment/project/tenant. See comments in the file for more detailed information on how to fill these in. To utilize OSP cinder storage for the registry add the following to `all.yml`
 ```
 openshift_hosted_registry_storage_kind: openstack
-openshift_hosted_registry_storage_access_modes: ['ReadWriteOnce']
```

Comment:
Please include `openshift_hosted_registry_storage_volume_size` here and explain that it can be used to set a proper size. (maybe with an indication that `20` is the default)